README.md

```diff
@@ -132,6 +132,22 @@ spec:
   type: managed
 ```
 
+## Resource tagging
```

Comment:
awesome, this covers the majority of https://issues.redhat.com/browse/INTLY-4687

Reply:
credit to you @aidenkeating  it's almost a straight copy of your demo.